packages/airnode-validator/src/commands/utils.ts

```diff
@@ -66,8 +66,9 @@ function getLatestPath(template: string): string | null {
  */
 export function getPath(template: string, messages: Log[], version = ''): string | null {
   if (version) {
-    if (fs.existsSync(path.resolve(validatorTemplatesPath, version, template))) {
-      return path.resolve(validatorTemplatesPath, version, template);
+    const parsedVersion = version.replace(/[0-9]+$/, '0');
```

Comment:
I think it makes more sense to rename the template directories to omit the patch version and ignore it instead of arbitrarily adding 0. I think this would be quite confusing for the users.

Reply:
removed in ddd8229